lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -602,6 +603,10 @@ instance FromText a => FromHttpApiData (ApiT a) where
 instance ToText a => ToHttpApiData (ApiT a) where
     toUrlPiece = toText . getApiT
 
+instance MimeUnrender OctetStream PostExternalTransactionData where
+    mimeUnrender _ = pure . PostExternalTransactionData . BL.toStrict
+
+
```

Comment:
Too many blank lines here.

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#use-only-a-single-blank-line-between-top-level-definitions

Reply:
done

Reply:
It doesn't seems right that the `OctetStream` is hex-encoded here. The binary data would be better off raw and untouched :+1: